main.go

```diff
@@ -88,13 +88,6 @@ func runTests(machine *client.RobotClient) {
 		return
 	}
 
-	if err := leftEncoder.ResetPosition(context.Background(), nil); err != nil {
-		logger.Error(err)
-	}
-	if err := rightEncoder.ResetPosition(context.Background(), nil); err != nil {
-		logger.Error(err)
-	}
-
```

Comment:
any reason for removing this? being unnecessary is fine

Reply:
unnecessary. now that I'm done testing the server is restarted every time the script is run so the ticks always start at 0